snmp/datadog_checks/snmp/data/default_profiles/aruba-cx-switch.yaml

```diff
@@ -0,0 +1,127 @@
+extends:
+  - aruba.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.47196.4.1.1.1.*
+  - 1.3.6.1.4.1.47196.4.1.1.1.1
+  - 1.3.6.1.4.1.47196.4.1.1.1.2
+  - 1.3.6.1.4.1.47196.4.1.1.1.3
+  - 1.3.6.1.4.1.47196.4.1.1.1.5
+  - 1.3.6.1.4.1.47196.4.1.1.1.50
+  - 1.3.6.1.4.1.47196.4.1.1.1.70
+  - 1.3.6.1.4.1.47196.4.1.1.1.101
+  - 1.3.6.1.4.1.47196.4.1.1.1.103
+  - 1.3.6.1.4.1.47196.4.1.1.1.104
+  - 1.3.6.1.4.1.47196.4.1.1.1.150
+  - 1.3.6.1.4.1.47196.4.1.1.1.304
+  - 1.3.6.1.4.1.47196.4.1.1.1.405
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.2.1.25.2.3.1.5.1
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.used
+      OID: 1.3.6.1.2.1.25.2.3.1.6.1
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.2.1.25.3.3.1.2.196608
```

Comment:
Same here, any idea why we are targeting index `196608` ?

Reply:
it's used to target the first core of the switch
https://community.arubanetworks.com/community-home/librarydocuments/viewdocument?DocumentKey=9094a953-1b18-424e-b690-d60d6a029377&CommunityKey=2fd943a6-8898-4dbe-915f-4f09e4d3c317&tab=librarydocuments

Reply:
👍

Reply:
Wait, but why only targeting the first core. Maybe we can collect cpu.usage from all cores and use average ?

Like this: https://github.com/DataDog/integrations-core/blob/e79c81a813356a2fdd558e5c9395a7d52b290f6d/snmp/datadog_checks/snmp/data/default_profiles/_cisco-cpu-memory.yaml#L4-L14

Reply:
I think depending on the switch the number of cores varies but we know that the minimum is one so we target the first core because we know that it's always there